CONTRIBUTORS

```diff
@@ -0,0 +1 @@
+See the list of contributors in the CONTRIBUTORS file in the organicmaps/organicmaps repository.
```

Comment:
There are different contributors in different code repositories, it is wrong to mix them. Github already provides a built-in way to see contributors. The same approach should be applied to the main repository, it will work out of the box and show everyone who contributed.
```suggestion
See the list of this repository's contributors at GitHub: https://github.com/organicmaps/website/graphs/contributors
```